### DISCUSSION TITLE
Recomendación para subir archivos

### DISCUSSION THREAD

**Discussion by AstorArico72** [AstorArico72/Proyecto-Transversal-1#11] [General]
### Para evitar posibles duplicados, tenemos que evitar subir todo el proyecto NetBeans.
No hay que subir ésto:

> Propiedades del proyecto. Sólo necesitamos un archivo para éso.
> Build.xml. Sólo necesitamos uno.
> El JAR de MariaDB. Sólo necesitamos uno, que se va a alojar en una carpeta de bibliotecas.
> La carpeta nbproject. Sólo necesitamos una.
> Manifest.mf. Para evitar duplicados.

En adición a eso, tenemos que asegurarnos de que los nombres de los paquetes coincidan. Dirigirse a la sección problemas para encontrar paquetes que no coinciden.
He propuesto algo en el hilo #10.

**Comment by PabloBarrionuevo** [comment]
Todos esos son autogenerados por el mismo IDE de NetBeans. (Excepto el jar de MariaDB)
